src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -124,7 +125,7 @@ export default {
 		},
 		checkoutRoute: {
 			type: String,
-			default: '#show-basket'
+			default: ''
```

Comment:
I think this was previously used to keep the loan card from redirecting. I realize we're now using a new loan card so it may not be needed any longer?

Reply:
I removed the redirect and added a @click function that would execute removal of the loan from the basket.  So the lend $25 button will be a toggle button that adds and removes loans from the basket.  This is all according to Sara's new design.

Reply:
Ok thanks for checking, this sounds like a new paradigm that we have not used on loan cards.

Reply:
How will one, "activate" the basket overlay if there is no longer a "checkout" button on the loan card or an automatic action to "show" the basket after adding a loan to it?

Reply:
There is a checkout button in the progress bar section when a campaign is active, but when the campaign is not active there's also the "basket" link at the top nav.  I mentioned that the progress section of the page would not be shown during inactive campaign periods, and Sara was okay with this because this is not the majority use case.  Most people will be checking out loans via campaign pages only when a campaign is active.  And even when the campaign is inactive, accessing the basket is not blocked, in fact it's always visible in the top right section of the page.

Reply:
It looks like the loan card now get the below button when ATB clicked, and clicking this button removes it from the basket. Is that the desired behavior?
![image](https://user-images.githubusercontent.com/16867161/231568323-a65c36b8-6435-4521-8626-91a252c500b4.png)

Reply:
Yes

Reply:
That's the whole point of this PR and aligns with Sara's new design for the page.